config.json

```diff
@@ -1,6 +1,6 @@
 {
     "website_name": "Binary.com",
-    "display_languages": "EN,ID,RU,ES,FR,PT,DE,ZH_CN,PL,AR,ZH_TW,VI,IT",
+    "display_languages": "EN,ID,RU,ES,FR,PT,DE,ZH_CN,PL,AR,ZH_TW,VI",
```

Comment:
This resulted from uploading an old version of config.json. Changed back in the most recent commit.